modules/visual_script/visual_script_builtin_funcs.h

```diff
@@ -101,6 +101,7 @@ class VisualScriptBuiltinFunc : public VisualScriptNode {
 		VAR_TO_BYTES,
 		BYTES_TO_VAR,
 		COLORN,
+		MATH_SMOOTHSTEP,
```

Comment:
Is it a problem if it's included further up before `MATH_DECTIME`, as for `core/math`?

Reply:
by @vnen
 > I believe this breaks compatibility if put here: it should be put at the end of the list. You can see in the doc changes that all constants got new values, so if someone is using a function from below, suddenly the functions will be changed and it'll break the code.